sale_crm_team_invoiced_payment_state/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 Tecnativa - Juan Carlos Oñate
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Crm Team Invoiced Payment State",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "category": "Sale",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/sale-workflow",
+    "depends": ["sale", "crm"],
```

Comment:
Everything depending on is in `sale` module, not in `crm`:

```suggestion
    "depends": ["sale"],
```

Reply:
https://github.com/OCA/sale-workflow/pull/3582